lab_1/main.py

```diff
@@ -74,10 +135,30 @@ def detect_language(unknown_profile: dict, profile_1: dict, profile_2: dict, top
     :param top_n: a number of the most common words
     :return: a language
     """
-    pass
+    if not isinstance(unknown_profile, dict):
+        return None
+    if not isinstance(profile_1, dict):
+        return None
+    if not isinstance(profile_2, dict):
+        return None
+    if not isinstance(top_n, int):
+        return None
+    compare_with_1 = compare_profiles(unknown_profile, profile_1, top_n)
+    compare_with_2 = compare_profiles(unknown_profile, profile_2, top_n)
+    lang = ""
+    if compare_with_1 > compare_with_2:
+        lang += profile_1["name"]
```

Comment:
I'm not sure why you need to add a language to an empty line

Reply:
Would it be better if there was an empty list instead of empty str?